bots/images/scripts/atomic.bootstrap

```diff
@@ -55,23 +55,17 @@ while [ $# -gt 2 ]; do
     shift
 done
 
-if [ -n "$TEST_DATA" -a -f "$TEST_DATA/$fragment" ]; then
-    echo Using "$TEST_DATA/$fragment"
-    cp "$TEST_DATA/$fragment" "$out"
+# we link to the file so wget can properly detect if we have already downloaded it
+# note that due to mirroring, timestamp comparison can result in unnecessary downloading
+out_base=`dirname $out`
+intermediate="$out_base/$fragment"
+
+if [ "$intermediate" != "$out" ]; then
+    wget --no-clobber --directory-prefix=$out_base $base/$fragment
+    cp $intermediate $out
 else
-
-    # we link to the file so wget can properly detect if we have already downloaded it
-    # note that due to mirroring, timestamp comparison can result in unnecessary downloading
-    out_base=`dirname $out`
-    intermediate="$out_base/$fragment"
-
-    if [ "$intermediate" != "$out" ]; then
-        wget --no-clobber --directory-prefix=$out_base $base/$fragment
-        cp $intermediate $out
-    else
-        rm -f $out
-        wget --directory-prefix=$out_base $base/$fragment
-    fi
+    rm -f $out
```

Comment:
These unquoted variables make me cringe :crying_cat_face:  (Not your fault, it's been like that forever)

Reply:
I'll take the chance to fix that.